src/cogs/owner.py

```diff
@@ -661,6 +686,10 @@ async def load_editor_tiles(self):
             inactive_x = int(c_x)
             inactive_y = int(c_y)
             tiling = int(tiling)
+            if tiling == TilingMode.TILING:
+                # Check for diagonal tiling
+                if pathlib.Path(f"data/sprites/{constants.BABA_WORLD}/{sprite}_16_1.png").exists():
+                    tiling = +TilingMode.DIAGONAL_TILING
```

Comment:
Copy pasted check from earlier?

Reply:
i think so

Reply:
Is there anything that reasonably can or should be done about this?

Reply:
in all honestly this is likely fine to just leave as is

Reply:
this was copied from convert_json.py which no longer exists so :P